nss-day-cohort-48/rare-client-orchesta-outlaws
# SLMfrontend issue 21 22 23

#### Changes Made


1. Added file `CommentProvider.js``CommentForm.js`  `CommentList.js`  to `comment` directory.
2. Basic CRUD Functionality.



#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout slm-frontend-issue-21-22-23

   ```
2. Ensure the backend and client are both running.

4. Test app functionality.

   > Example: When on post details try viewing all comments, adding comments, then deleting said comment. NOTE: doesn't currently redirect back to post/details.
   
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
